includes/discovery/os/linux.inc.php

```diff
@@ -50,6 +50,20 @@
                 $os = 'airos-af';
             }
         }
+        else if (trim(snmp_get($device, 'GANDI-MIB::rxCounter.0', '-Osqnv', 'GANDI-MIB')) != '') {
```

Comment:
I have been preferring:
`else if (snmp_get($device, 'GANDI-MIB::rxCounter.0', '-Osqnv', 'GANDI-MIB') !== false) {`

Other than that minor preference :+1: